### ISSUE TITLE
yii3-progress page giving error 502

### ISSUE THREAD

**Issue by atrandafir** [yiisoft-contrib/yiiframework.com#965]
https://www.yiiframework.com/yii3-progress

![image](https://user-images.githubusercontent.com/3097251/148415591-2a3acbf9-798b-4b21-9e87-63cc6bd406bc.png)

**Comment by samdark** [comment]
Yeah. Information should be collected via cron instead of the web thread.

**Comment by samdark** [comment]
If you'll do refresh it should be OK.
